qiskit/algorithms/optimizers/spsa.py

```diff
@@ -190,6 +192,9 @@ def __init__(
             callback: A callback function passed information in each iteration step. The
                 information is, in this order: the number of function evaluations, the parameters,
                 the function value, the stepsize, whether the step was accepted.
+            termination_callback: A callback function executed at the end of each iteration step. The
+                arguments are, in this order: current parameters, estimate of the objective
```

Comment:
I think it would be important to mention how the objective function is estimated 🙂